static-assets/components/cstudio-forms/controls/rte.js

```diff
@@ -517,9 +449,33 @@ CStudioAuthoring.Module.requireModule(
               } else {
                 _thisControl.editor.plugins.craftercms_paste_extension?.paste_postprocess(plugin, args);
               }
-            }
+            },
+            ...(rteConfig?.tinymceOptions && {
+              ...CrafterCMSNext.util.object.reversePluckProps(
+                rteConfig.tinymceOptions,
+                'target', // Target can't be changed
+                'inline', // The control will always have the default (false) in forms-engine.
+                'setup',
+                'base_url',
+                'encoding',
+                'autosave_ask_before_unload', // Autosave options are removed since it is not supported in control.
+                'autosave_interval',
+                'autosave_prefix',
+                'autosave_restore_when_empty',
+                'autosave_retention',
+                'file_picker_callback', // File picker integration with our data sources is already implemented in the control
+                'height', // Height is set via control properties
+                'width', // Width is set via control properties
+                'paste_postprocess', // Already implemented for paste and drag&drop using our data sources.
+                'images_upload_handler', // Images upload integration with our data sources is already implemented in the control
+                'code_editor_inline', // Code editor will always be inline in forms-engine.
+                'plugins', // Considered/used above, mixed with our options
+                'toolbar_sticky', // Toolbar is configured and styled to be sticky in forms-engine
+                'relative_urls' // To avoid allowing convertion of urls to be relative to the document_base_url
+              )
+            })
           });
-          _thisControl.editor = editor;
+          // _thisControl.editor = editor;
```

Comment:
?

Reply:
Removed, this was incorrectly set (editor variable was wrong) and overridden after the tinymce initialization